$OpenBSD$

Fix evry_plug_windows.c:171: warning: comparison between signed and unsigned.

--- src/modules/everything/evry_types.h.orig	Tue Nov  8 16:34:40 2011
+++ src/modules/everything/evry_types.h	Tue Nov  8 16:35:06 2011
@@ -228,7 +228,7 @@ struct _Plugin_Config
 
   /* minimum input char to start query items,
      this must be handled by plugin */
-  int min_query;
+  unsigned int min_query;
 
   /* show items of plugin in aggregator */
   int aggregate;
